sycl/test/functor/kernel_functor.cpp

```diff
@@ -18,27 +19,6 @@ constexpr auto sycl_read_write = cl::sycl::access::mode::read_write;
 constexpr auto sycl_global_buffer = cl::sycl::access::target::global_buffer;
 
 // Case 1:
-// - functor class is defined in an anonymous namespace
-// - the '()' operator:
-//   * does not have parameters (to be used in 'single_task').
-//   * has the 'const' qualifier
```

Comment:
Is this change for both SYCL-1.2.1 and SYCL-2020 or only for SYCL-2020?
If the change is only for SYCL-2020, I'd prefer splitting the test, so that testing of SYCL-1.2.1 functionality isn't impacted.

Reply:
This applies for SYCL 1.2.1 as well as SYCL 2020